content/_admin/ip-whitelisting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Egress IP Addresses & IP Whitelisting
+layout: article
+section: elastic.io Specific Settings
+description: "Document lists public IP addresses of our clusters to whitelist if required."
+order: 1
+category: support
+---
+
+**Summary**
+  * *Traffic from the {{site.data.tenant.name}} platform comes from fixed IP addresses*
+  * *The fixed IP addresses are listed on this page*
+  * *{{site.data.tenant.companyName}} commits to supporting the ability to allowing integrators to add {{site.data.tenant.name}} IP addresses to their whitelists*
+<hr />
```

Comment:
This is pure HTML approach, not a problem really. In Markdown you can use just `---`